scripts/src/javatests/org/oppia/android/scripts/docs/KDocValidityCheckTest.kt

```diff
@@ -154,6 +272,22 @@ class KDocCheckTest {
     assertThat(outContent.toString().trim()).isEqualTo(KDOC_CHECK_PASSED_OUTPUT_INDICATOR)
   }
 
+  @Test
+  fun testKDoc_injectPrimaryConstructor_noSpecialBehaviour_checkShouldPass() {
+    val testContent =
+      """
+      /** Test Class. */
+      class SelectionFragmentModel @Inject constructor(){
```

Comment:
```suggestion
      class SelectionFragmentModel @Inject constructor() {
```

Reply:
Done.

Reply:
Was this addressed? The latest code version doesn't seem to include the space.

Reply:
Apologies! Just missed this. Now updated.